ext/opentelemetry-ext-flask/src/opentelemetry/ext/flask/__init__.py

```diff
@@ -60,7 +61,9 @@ def _before_flask_request():
         otel_wsgi.get_header_from_environ, environ
     )
 
-    tracer = trace.tracer()
+    tracer = trace.tracer_source().get_tracer(
+        "opentelemetry-ext-flask", __version__
```

Comment:
That would be another possibility. There should probably a unique mapping `full module name -> package` anyway.

Reply:
I changed it to the module name in e4db2174fcd04d53efd449f13e149059f5215e7b. But note that once we have a registry pattern, we will be creating a tracer per module instead of per library with the most naive implementation at least.

Reply:
Yeah, one risk of copying `logging` here is that users might reasonably assume these names are meant to be hierarchical too.